frontend/src/app/HeaderTools.tsx

```diff
@@ -86,62 +83,64 @@ const HeaderTools: React.FC<HeaderToolsProps> = ({ onNotificationsClick }) => {
       <DropdownItem
         key="community"
         onClick={handleHelpClick}
-        className="odh-dashboard__external-link"
         href={COMMUNITY_LINK}
         target="_blank"
         rel="noopener noreferrer"
       >
-        Community
-        <ExternalLinkAltIcon />
+        Community <ExternalLinkAltIcon />
       </DropdownItem>,
     );
   }
 
   return (
-    <PageHeaderTools>
-      <PageHeaderToolsGroup className="hidden-xs">
-        {!dashboardConfig.spec.dashboardConfig.disableAppLauncher ? <AppLauncher /> : null}
-        <PageHeaderToolsItem>
-          <NotificationBadge isRead count={newNotifications} onClick={onNotificationsClick} />
-        </PageHeaderToolsItem>
-        {helpMenuItems.length > 0 ? (
-          <PageHeaderToolsItem>
-            <Dropdown
-              removeFindDomNode
-              position={DropdownPosition.right}
-              toggle={
-                <DropdownToggle
-                  id="toggle-id"
-                  onToggle={() => setHelpMenuOpen(!helpMenuOpen)}
-                  toggleIndicator={CaretDownIcon}
-                >
-                  <QuestionCircleIcon />
-                </DropdownToggle>
-              }
-              isOpen={helpMenuOpen}
-              dropdownItems={helpMenuItems}
-            />
-          </PageHeaderToolsItem>
-        ) : null}
-        <PageHeaderToolsItem>
+    <Toolbar isFullHeight isStatic>
```

Comment:
What does `isStatic` do?

Reply:
You are right, I just copy paste it from PF demo, there should be no difference even if we remove the `isStatic`, I will remove it.